pygments/lexers/arturo.py

```diff
@@ -0,0 +1,283 @@
+"""
+    pygments.lexers.arturo
+    ~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the Arturo language.
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments.lexer import bygroups, default, DelegatingLexer, do_insertions,\
+    include, RegexLexer, this, using, words
+from pygments.token import Comment, Generic, Keyword, Name, Number, Operator,\
+    Other, Punctuation, String, Text
+
+from pygments.util import ClassNotFound, get_bool_opt
+
+__all__ = [
+    'ArturoLexer'
+]
+class ArturoLexer(RegexLexer):
+    """
+    For Arturo source code
+
+    .. versionadded:: 0.9.80
```

Comment:
should be the (next) Pygments version - 2.14.0

Reply:
Ooh! Sorry, got it! My bad! 😂
I added current Arturo version.

Reply:
- [x] [change versionadded](https://github.com/pygments/pygments/pull/2259/commits/67b45a9aa8a5a893ce9a7aa3edf163c49bf94b07)